### ISSUE TITLE
Hadolint fails in the FROM when specifying a port number for a registry

### ISSUE THREAD

**Issue by Rickkwa** [hadolint/hadolint#169]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior

The linter should run on Dockerfiles that use a custom registry with a port number.

### Actual behavior

The linter errors out before running any linting checks.

### Steps to reproduce the behavior

Create the following Dockerfile:

```
FROM myregistry.com:5000/imagename:tag
```

Run `docker run --rm -i hadolint/hadolint < Dockerfile`.

You will see an error like:

```
/dev/stdin:1:35 unexpected ':' expecting spaces, "AS" or a new line followed by the next instruction
```

If you remove the `:5000`, the error goes away.

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
# docker run --rm hadolint/hadolint hadolint --version
Haskell Dockerfile Linter v1.5.0-9-g508b1cf
```

**Dockerfile** (if relevant)

```Dockerfile
FROM myregistry.com:5000/imagename:tag
```

**Additional environment details (OS, stack version, etc.)**

```
$ docker version
Client:
 Version:      17.05.0-ce
 API version:  1.29
 Go version:   go1.7.5
 Git commit:   89658be
 Built:        Thu May  4 22:06:25 2017
 OS/Arch:      linux/amd64

Server:
 Version:      17.05.0-ce
 API version:  1.29 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   89658be
 Built:        Thu May  4 22:06:25 2017
 OS/Arch:      linux/amd64
 Experimental: false
```

**Comment by Rickkwa** [comment]
This is an issue of having 2 colons.

**Comment by zemanlx** [comment]
Good catch @Rickkwa!

### Note to ourself
What else we can expect in pattern **registry**/`imagename`:_label_
- **localhost:5000**/`debian`:_9_
- **172.30.25.243:5000**/`lgx-project/ruby-ex`:_3_

in image tag area we can expect that there won't be a colon [docker_registry/tags.py#L25](https://github.com/docker/docker-registry/blob/f93b432d3fc7befa508ab27a590e6d0f78c86401/docker_registry/tags.py#L25)
and I have also found a word description of naming at https://docs.docker.com/engine/reference/commandline/tag/#extended-description
